src/components/playlist.tsx

```
@@ -39,32 +40,52 @@ export default function Playlist({ setIsOpen, setCurrentTrack }: Props) {
                 key={index}
-                className="mx-2 my-3 w-full cursor-pointer rounded border-b border-gray-100 p-3 text-slate-500 transition-colors duration-300 hover:bg-gray-50 hover:text-slate-800"
-                onClick={() => handlePlylistItemClick(index)}
+                className="mx-2 my-3 w-full rounded border-b border-gray-100 p-3 text-slate-500 transition-colors duration-300 hover:bg-gray-50 hover:text-slate-800"
               >
-                <div className="flex items-center">
-                  <span className="m-2 flex size-8 items-center justify-center rounded-full bg-gray-100 text-xs font-medium">
-                    {index + 1}
+                <div className="flex items-center gap-3">
+                  <span
+                    onClick={() => handlePlylistItemClick(index)}
+                    className="flex cursor-pointer flex-1 items-center"
```

ESLint

[error] Replace `cursor-pointer·flex-1` with `flex-1·cursor-pointer`

(prettier/prettier)